mixnet/client/src/config.rs

```diff
@@ -1,10 +1,33 @@
 use std::net::SocketAddr;
 
+use futures::Sink;
 use mixnet_topology::MixnetTopology;
 use serde::{Deserialize, Serialize};
 
+use crate::receiver::Receiver;
+
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct MixnetClientConfig {
-    pub listen_addr: SocketAddr,
+    pub mode: MixnetClientMode,
     pub topology: MixnetTopology,
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub enum MixnetClientMode {
+    Sender,
+    SenderReceiver(SocketAddr),
```

Comment:
I will change the mixclient to use websocket to receive packets from mixnode, instead of having a listen_address.

Reply:
I’m converting this PR as draft.

Reply:
I’ll do this at the next PR to not make this PR bigger. Now this PR works